index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My portfolio</title>
+    <!-- Google Icons --> 
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div id="root">
+        <header class="top_header">
```

Comment:
- [ ] Hi, Please I am reading your project requirements, and it says 2 sections need to be built for this Project. But you rather built it in 1 section which is `<header>`. Please check the link: [Project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md)

Reply:
- [ ] Please implement the changes required by the previous reviewer. Your project is to have two sections, a `<header>` and a `<section>`. 

- The `<header>` should have the `nav` with your `logo` and `harmburger icon`
- The `<section>` should hold the contents about you.